server/metrics/tracing.go

```diff
@@ -22,9 +22,8 @@ import (
 )
 
 const (
-	GrpcTracingServiceName      = "tigris.grpc"
-	KvTracingServiceName        = "tigris.fdb.kv"
-	TxManagerTracingServiceName = "tigris.tx.manager"
+	KvTracingServiceName        = "tigris-server.fdb.kv"
```

Comment:
Nice job adding these to `const`! Should the service here simply be called `fdb`?